core/base/compactTriangulation/CompactTriangulation.cpp

```diff
@@ -0,0 +1,30 @@
+#include <CompactTriangulation.h>
+
+using namespace std;
+using namespace ttk;
+
+CompactTriangulation::CompactTriangulation() {
+  setDebugMsgPrefix("CompactTriangulation");
+  clear();
+
+#ifdef TTK_ENABLE_OPENMP
```

Comment:
this distinction is not necessary.
`threadNumber_` is initialized to 1 when `TTK_ENABLE_OPENMP` is not defined.
(see https://github.com/topology-tool-kit/ttk/blob/dev/core/base/common/BaseClass.cpp)

Reply:
please update.

Reply:
solved.